packages/trader/src/App/Components/Elements/Media/media-icon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type TMediaIcon = {
+    disabled: React.SVGAttributes<SVGElement>;
+    enabled: React.SVGAttributes<SVGElement>;
```

Comment:
```suggestion
    disabled: React.FC<React.SVGAttributes<SVGElement>>;
    enabled: React.FC<React.SVGAttributes<SVGElement>>;
```

Reply:
@farzin-deriv Wouldn't be a problem, than in file svg.d.ts enabled/disabled were define as ⬇️ ?
```
declare module '*.svg' {
    const content: React.SVGAttributes<SVGElement>;
    export default content;
}
```

Reply:
@kate-deriv Actually it seems like the type in `svg.d.ts` is wrong, I believe @henry-deriv already fixed it in one of his PRs, If you want you can also add the fix in your PR, Please double check with @henry-deriv 🙇🏻

Reply:
I'll ask him, thank you

Reply:
I have this in my change. You are welcome to include the exact same change. I dont see an issue

Reply:
Done, thank you